xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -122,7 +122,7 @@ This constructor assumes that the entire JSON payload is contained in `jsonData`
         <param name="jsonData">The UTF-8 encoded JSON text to process.</param>
         <param name="isFinalBlock">
           <see langword="true" /> if the input sequence contains the entire data to process; <see langword="false" /> only the input span contains partial data with more data to follow.</param>
```

Comment:
boolean params have a special entry in the writing guidelines: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors

maybe something like this?

```suggestion
          <see langword="true" /> to indicate that the input sequence contains the entire data to process; <see langword="false" /> to indicate the input span contains partial data with more data to follow.</param>
```